tensorboard/webapp/routes/dashboard_deeplink_provider_test.ts

```diff
@@ -358,10 +367,14 @@ describe('core deeplink provider', () => {
     store.overrideSelector(selectors.getOverriddenFeatureFlags, {
       enabledExperimentalPlugins: ['foo', 'bar', 'baz'],
     });
+    store.overrideSelector(selectors.getPluginQueryParams, {
```

Comment:
Does it make sense for this to live in 'serializes feature flags'? Should it be its own test?

Reply:
I think it makes sense here because these are (were?) query parameters and they are being serialized.
If you feel strongly about it I'm happy to make another test.

Reply:
I don't understand what the additions to this test have to do with 'feature flags'. They don't seem necessary.

The comment for the test:
```
  /**
   * One test to verify that feature flags are correctly serialized using
   * featureFlagsToSerializableQueryParams.
   */
```

The name for the test:
``` 
it('serializes feature flags', () => {
```

Reply:
I fully read "feature flags" as "query params". I guess working on TB has created an alias for me